exercise.wwwapi/Endpoints/BookEndpoints.cs

```diff
@@ -0,0 +1,47 @@
+﻿using exercise.wwwapi.Models;
+using exercise.wwwapi.Repository;
+using Microsoft.AspNetCore.Mvc;
+
+namespace exercise.wwwapi.Endpoints
+{
+    public static class BookEndpoints
+    {
+        public static void ConfigureBookEndpoint(this WebApplication app)
+        {
+            var bookGroup = app.MapGroup("Book");
+
+            bookGroup.MapGet("/books", GetBooks);
+            bookGroup.MapGet("/books{id}", GetBook);
+            bookGroup.MapDelete("/books{id}", DeleteBook);
```

Comment:
do you need books in the URLs?? and how would that url look when you test it?

DELETE method
```
localhost:1234/book/books1
```

Reply:
should be fixed now